### DISCUSSION TITLE
why does this keep happening?

### DISCUSSION THREAD

**Discussion by 0s9df8c0sdfd0** [valinet/ExplorerPatcher#2805] [General]
so im on windows 11 22h2 and when i start my pc it gives me a error it failed to start three times even when theres no update for explorerpatcher and everything is the same like what i did with explorerpatcher. how to fix?

**Comment by pyrates999** [comment]
Uninstall EP and download latest release and install that.  You may have an old version installed.

If that doesn't work.  Open up EP Properties application and go to **about** and click on **restore default settings**, and reboot.

Please post windows version and build.  You can find this by running winver.

Please include any other software you have installed that modifies the UI of windows.

**Reply by 0s9df8c0sdfd0** [reply]
I use Windows 11 Home Edition 22H2 OS Build 22621.3007 

And it should be the latest version. of Explorer Patcher.

And most of it, I used Registry Editor to make some hacks to make it look like windows 10. I used Explorer Patcher for the taskbar and start menu and other stuff. I will reply again if it works or not.

**Comment by 0s9df8c0sdfd0** [comment]
Also it works. I restored default settings and changed it to what i want, And it worked.

When I restarted my PC And restart it, It works. No error.

**Comment by pyrates999** [comment]
Don't modify the registry directly unless you know what you are doing.

**Comment by 0s9df8c0sdfd0** [comment]
I know what I'm doing with the Registry Editor. And I just used the
Registry Editor for custom file icons when i make a new one, And i disabled
the Windows 10X Startup ring, And i removed the Snap Assist Flyout. I
didn't do more stuff than those I just told you.

**Comment by pyrates999** [comment]
Ok.  I wasn't aware you could disable the bootup animation.
